joint_trajectory_controller/test/test_trajectory_controller.cpp

```diff
@@ -1104,6 +1104,17 @@ INSTANTIATE_TEST_CASE_P(
       std::vector<std::string>({"position", "velocity", "acceleration"}),
       std::vector<std::string>({"position", "velocity", "acceleration"}))));
 
+// only velocity controller
```

Comment:
It would also be cool to add a dedicated test for parameters. It can be parameterized and then depending on interfaces checked. See `test_joint_trajectory_controller_utils.hpp` file (or similar name).

Reply:
What do we want to test here? Parameter propagation from overrides (e.g. yaml files) or we want to add parametrized test that will receive joint_names, interfaces, etc (this one already exists, only the joint names are hardcoded).

Reply:
We want to run all existing test on this. Not only propagation, but calculations and setup in general.